### DISCUSSION TITLE
After uploading the image, it cannot be recognized and remains stuck in the content generation state.

### DISCUSSION THREAD

**Discussion by cjnss008** [open-webui/open-webui#6866] [Issues]
![1](https://github.com/user-attachments/assets/06e7968a-90c8-4bc2-92ef-f782fafa37c8)
After uploading the image, it cannot be recognized and remains stuck in the content generation state.
The Docker logs are empty. After refreshing the page, this message will no longer be displayed.
Uploading other files, such as docx files, works fine.
Installed using Docker.

**Comment by marco-cinalli** [comment]
I witness a similar issue. Everytime, I either upload an image or I let open-webui generate an image, I cannot input any more prompts. Open webui seems to get stuck, I have to create a new prompt. This happens only when I use open-webui with API (it does not happen if I run ollama locally). Could it be because open webui is trying to send through API the full image in base64 encoding and that is not accepted? (that is my assumption, but I don't know how to solve)
